apps/aot/src/app/events/[year]/[day]/submissions/[[...catchAll]]/_components/suggestions.tsx

```diff
@@ -5,6 +5,7 @@ interface SuggestionsProps {
   challengeId: number;
 }
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 export function Suggestions({ track, challengeId }: SuggestionsProps) {
```

Comment:
[Question] Can this unused component be deleted?

Btw, if you haven't tried it out yet, I'd definitely recommend [Knip](https://knip.dev) as an awesome tool to detect unused file exports and project dependencies.

Reply:
🔪